dockers/agent/core/faiss/Dockerfile

```diff
@@ -37,6 +37,8 @@ ENV REPO vald
 ENV PKG agent/core/faiss
 ENV PKG_INTERNAL agent/internal
 ENV APP_NAME faiss
+ENV AARCH aarch_64
+ENV XARCH x86_64
```

Comment:
Do we need these?

Reply:
I forgot to delete it... Thank you for catching it 🙏 
https://github.com/vdaas/vald/pull/2334/commits/772eb6680ba9ba129b113edf4717bed827ac8e64